Core/GDCore/IDE/Events/EventsVariablesFinder.cpp

```diff
@@ -203,6 +207,32 @@ std::set<gd::String> EventsVariablesFinder::FindArgumentsInInstructions(
   return results;
 }
 
+std::set<gd::String> EventsVariablesFinder::FindArgumentsInEventsAndDependencies(
+    const gd::Platform& platform,
+    const gd::Project& project,
+    const gd::Layout& layout,
+    const gd::EventsList& events,

```

Comment:
I think in this method, we should remove `events` argument and automatically use `layout.GetEvents()`. Why? Because we use DependenciesAnalyzer, which will analyse the events of the whole scene (so it would be inconsistent if we don't pass the whole layout events as argument).